packages/api/src/routes/internal/medical/patient-job.ts

```diff
@@ -51,6 +143,52 @@ router.post(
   })
 );
 
+const failOrCancelSchema = z.object({
+  reason: z.string(),
+});
+
+/**
+ * POST /internal/patient/job/:jobId/fail
```

Comment:
In which situation is this called? I don't see that on the code.

I'd have a `POST .../job/:jobId` with the status being a parameter, like we have on the bulk import job.